files/build_templates/packages.json.j2

```diff
@@ -1,6 +1,6 @@
 {
 {% for docker_installation_target in installer_images.strip().split() -%}
-{% set pkgname, docker_build_path, image = docker_installation_target.split('|') -%}
+{% set pkgname, docker_build_path, _, image = docker_installation_target.split('|') -%}
```

Comment:
Thanks for the fix. I feel we need extract the **machine** and compare **machine** with $TARGET_MACHINE in j2 syntax in packages.json.j2. You could pass the TARGET_MACHINE as a environment variable when j2 command for generating packages.json is invoked. 
```
MACHINE=$TARGET_MACHINE  j2 $BUILD_TEMPLATES/packages.json.j2 | sudo tee $FILESYSTEM_ROOT/$SONIC_PACKAGE_MANAGER_FOLDER/packages.json
```

Reply:
@judyjoseph Thanks for suggestion, I am not really familiar with new concept of machine. I'll investigate and reply.